### ISSUE TITLE
Cannot install patchouli dependency

### ISSUE THREAD

**Issue by nikjust** [ClothCreators/more-apis-support#1]
Error pastebin
https://pastebin.com/rEd8XrQK

**Comment by CrispyChips6660** [comment]
wait, that shouldn't be happening
I'll fix it ASAP

**Comment by CrispyChips6660** [comment]
are you using the latest version?

**Comment by nikjust** [comment]
> are you using the latest version?

Yes, Mcreator 2020.4 latest, more apis 0.5

**Comment by CrispyChips6660** [comment]
ok then, I'll check it
It may be a bad internet or your country blocking the website that you download patchouli from though
can you access [this](https://maven.blamejared.com/) website?

**Comment by nikjust** [comment]
> ok then, I'll check it
> It may be a bad internet or your country blocking the website that you download patchouli from though
> can you access [this](https://maven.blamejared.com/) website?

Yes, i can

**Comment by nikjust** [comment]
internet:
Download:94.88
Upload:95.32
Ping:1 ms
loss: 0%

**Comment by nikjust** [comment]
Can help replacing depencies version like this:

```
dependencies {
      //Patchouli
        compileOnly fg.deobf("vazkii.patchouli:Patchouli:1.14.4-1.1-26.5:api")
        runtimeOnly fg.deobf("vazkii.patchouli:Patchouli:1.14.4-1.1-26.5")
      }
```

P.S i don't know how compile plugin file

**Comment by nikjust** [comment]
> Can help replacing depencies version like this:
> 
> ```
> dependencies {
>       //Patchouli
>         compileOnly fg.deobf("vazkii.patchouli:Patchouli:1.14.4-1.1-26.5:api")
>         runtimeOnly fg.deobf("vazkii.patchouli:Patchouli:1.14.4-1.1-26.5")
>       }
> ```
> 
> P.S i don't know how compile plugin file

Whoops! i forgot the question mark

**Comment by nikjust** [comment]
Stacktrace + Debug:
https://sites.google.com/view/debug-code-patchouli

**Comment by CrispyChips6660** [comment]
[this is what I did](https://github.com/ClothCreators/more-apis-support/blob/master/src/apis/patchouli.yaml)

**Comment by CrispyChips6660** [comment]
Nope, this is definitely your internet's problem, just added patchouli dep and it worked

**Comment by CrispyChips6660** [comment]
[Fixed this in 0.5.1 :)](https://github.com/ClothCreators/more-apis-support/releases/tag/v0.5.1) @nikjust
